### ISSUE TITLE
script fails line 424 systemd-resolve: unrecognized option '--flush-caches'

### ISSUE THREAD

**Issue by pakair** [jonathanio/update-systemd-resolved#72]
I'm on Linux Mint 18.3 
```
sudo openvpn --config pavlos.ovpn I get ...
systemd-resolve: unrecognized option '--flush-caches'

```

**Comment by Edu4rdSHL** [comment]
What systemd version?

**Comment by pakair** [comment]
On Mint 18.3 systemd --version 
systemd 229
+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ -LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN

**Comment by Edu4rdSHL** [comment]
Current systemd version is 238, you should update your system.

**Comment by piotr-dobrogost** [comment]
Just revert the change made in https://github.com/jonathanio/update-systemd-resolved/pull/63 and see https://github.com/jonathanio/update-systemd-resolved/issues/62 for more details.

**Comment by Edu4rdSHL** [comment]
Doesn't make sense a revert because an outdated version of systemd.

Instead we can just evaluate if the --flush-caches is sucess or not and print some warning.

**Comment by tomeon** [comment]
Fixed in #99.

**Comment by Edu4rdSHL** [comment]
So far:

```
edu4rdshl ~ 
❯ busctl call FlushCaches
Too few arguments.
```
So, we now depend on a specific version of dbus?

**Comment by tomeon** [comment]
@Edu4rdSHL -- It's not `busctl call FlushCaches`, it's `busctl_call FlushCaches`, where `busctl_call` is [this function](https://github.com/cmadamsgit/update-systemd-resolved/blob/d2f5ee51fe531ebd473fa4550dafe7db81f939fa/update-systemd-resolved#L47-L54):

```shell
busctl_call() {
  # Preserve busctl's exit status
  busctl call "$DBUS_DEST" "$DBUS_NODE" "${DBUS_DEST}.Manager" "$@" || {
    local -i status=$?
    emerg "'busctl' exited with status $status"
    return $status
  }
}
```

And `DBUS_DEST` and `DBUS_NODE` are defined [here](https://github.com/cmadamsgit/update-systemd-resolved/blob/d2f5ee51fe531ebd473fa4550dafe7db81f939fa/update-systemd-resolved#L28-L30):

```shell
# Define what needs to be called via DBus
DBUS_DEST="org.freedesktop.resolve1"
DBUS_NODE="/org/freedesktop/resolve1"
```

So the full `busctl` command line is:

```shell-session
$ busctl call org.freedesktop.resolve1 /org/freedesktop/resolve1 org.freedesktop.resolve1.Manager FlushCaches
```
